src/System.Private.CoreLib/src/System/Reflection/Emit/FieldToken.cs

```diff
@@ -2,84 +2,35 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*============================================================
-**
-** 
-** 
-**
-**
-** Purpose: Represents a Field to the ILGenerator Class
-**
-** 
-===========================================================*/
-
-using System;
-using System.Reflection;
-
 namespace System.Reflection.Emit
 {
-    // The FieldToken class is an opaque representation of the Token returned
-    // by the Metadata to represent the field.  FieldTokens are generated by 
-    // Module.GetFieldToken().  There are no meaningful accessors on this class,
-    // but it can be passed to ILGenerator which understands it's internals.
+    /// <summary>
+    /// The FieldToken class is an opaque representation of the Token returned
+    /// by the Metadata to represent the field.  FieldTokens are generated by
+    /// Module.GetFieldToken().  There are no meaningful accessors on this class,
+    /// but it can be passed to ILGenerator which understands it's internals.
+    /// </summary>
     public struct FieldToken
     {
         public static readonly FieldToken Empty = new FieldToken();
-
-        internal int m_fieldTok;
-        internal object m_class;
-
-        // Creates an empty FieldToken.  A publicly visible constructor so that
-        // it can be created on the stack.
-        //public FieldToken() {
-        //    m_fieldTok=0;
-        //    m_attributes=0;
-        //    m_class=null;
-        //}
-        // The actual constructor.  Sets the field, attributes and class
-        // variables
+        private object _class;
 
         internal FieldToken(int field, Type fieldClass)
         {
-            m_fieldTok = field;
-            m_class = fieldClass;
+            Token = field;
+            _class = fieldClass;
         }
 
-        public int Token
-        {
-            get { return m_fieldTok; }
-        }
+        public int Token { get; }
```

Comment:
It would be better to keep this as an explicit field. Hiding state in implicit properties makes things harder to understand.

Reply:
The VS code editor presented this as a suggestion and grayed out the field. It is my understanding is that this is a (new) best practice to convert fields that are only written to once into an autoprop?

Reply:
I do not think that this is a uniformly good practice.  

The auto-properties make sense when:
- The object state is not that important given the nature of the project. We put a lot of emphasis on the state in coreclr/corefx by having a guideline to put it first, use underscores to refer to it, etc.
- And the code is not performance sensitive. Auto-properties produce bigger, potentially slower IL.

Reply:
okay fair enough - reverted this change then